charts/kubewarden-controller/templates/webhooks.yaml

```diff
@@ -34,6 +74,7 @@ webhooks:
     - v1
     - v1beta1
   clientConfig:
+    caBundle: {{ $caCert }}
```

Comment:
(opening thread here randomly)

There seems to be missing the clientConfig.caBundle for the admissionReviewVersions with name  `madmissionpolicy.kb.io`, currently line 98.